lib/judge/config.rb

```diff
@@ -8,24 +8,24 @@ class Config
     @@ignore_unsupported_validators = false
 
     def expose(klass, *attributes)
-      attrs = (@@exposed[klass] ||= [])
+      attrs = (@@exposed[klass.name] ||= [])
       attrs.concat(attributes).uniq!
     end
 
     def exposed
-      @@exposed
+      @@exposed.map { |k, v| [k.constantize, v] }.to_h
```

Comment:
@hgani It looks like you added this to get some tests to pass, but I'm thinking it may be the tests that need to be adjusted. First, it looks like Ruby 1.9.3 doesn't like the usage of `to_h` here (whether or not we should be still supporting 1.9.3 is another topic). When I reverted this change and updated the tests to use a stringed key, instead of class, they passed.

```ruby
Judge.config.exposed['User'].should eql [:username, :email]
``` 

I don't think `Judge.config.exposed` is part of the documented external API and the only usage internal.

@joecorcoran Any opinions here?